### ISSUE TITLE
Data

### ISSUE THREAD

**Issue by meanbeanmachine** [richzimmerman/DFCClientClone#1]
Do you have the data for the World?  i.e. the room text, monsters, classes, skills, etc.

**Comment by richzimmerman** [comment]
No, this client was built/wrapped around a forked version of CoffeeMUD, which was the game engine and contained all of that data. The client just received and displayed all the data sent from the game server.
